boards/arm/atsamr21_xpro/atsamr21_xpro.dts

```diff
@@ -91,7 +91,7 @@
 	#address-cells = <1>;
 	#size-cells = <0>;
 
-	cs-gpios = <&portb 31 0>;
+	cs-gpios = <&portb 31 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
```

Comment:
Should this be just GPIO_ACTIVE_LOW, i.e. push pull and inverted?

Reply:
@nandojve ^^

Reply:
Yep! Fixed